draft-ietf-modpod-group-processes.md

```diff
@@ -261,7 +261,9 @@ manage what may be difficult and contentious debates within their
 groups.  Within the bounds of this principle, it is left to
 moderators' judgement to determine when they must act, with the
 understanding that some situations may require fast responses.
-{{appeals}} discusses the handling of disagreements.
+{{appeals}} discusses the handling of disagreements.  In these
+circumstances, administrators may later rescind or amend moderator actions,
+informing them when they have done so.
```

Comment:
I think I'd want the chairs to enter a discussion with the moderators if this happens, and not just unilaterally letting them override a moderator team decision. The case I am worried about is unresponsive chairs, moderators acting because of that, and then the chairs reverting based on "this is OK in our WG", which would counter the desire to have uniform standards across the org.